main.tf

```diff
@@ -9,6 +9,18 @@ locals {
     enabled  = "${var.gitlab_runner_ssh_cidr_blocks}"
     disabled = "${list()}"
   }
+    // Convert list to a string separated and prepend by a comma
+    docker_machine_options_string = "${format(",%s", join(",", formatlist("%q", var.docker_machine_options)))}"
+
+    // Ensure off peak is optional
+    runners_off_peak_periods_string = "${var.runners_off_peak_periods == "" ? "" : format("OffPeakPeriods = %s", var.runners_off_peak_periods)}"
+
+    // Define key for runner token for SSM
+    secure_parameter_store_runner_token_key = "${var.environment}-${var.secure_parameter_store_runner_token_key}"
+
+    // custom names for instances and security groups
+    name_runner_instance = "${var.overrides["name_runner_agent_instance"] == "" ? local.tags["Name"] : var.overrides["name_runner_agent_instance"]}"
+    name_sg              = "${var.overrides["name_sg"] == "" ? local.tags["Name"] : var.overrides["name_sg"]}"
 }
 
 resource "aws_security_group" "runner" {
```

Comment:
I have just solved #62 which cause a change set every time running an apply. Therefore an extra security group  is introduced. Please can you sync with develop and updated the new security group with this change as well.

Reply:
Can you check if all is ok now?